cax/tasks/filesystem.py

```diff
@@ -145,15 +147,28 @@ def each_run(self):
             if 'host' not in data_doc or data_doc['host'] != config.get_hostname():
                 continue
                 
-            if self.node == data_doc['host'] and self.status == data_doc['status']:
+            if self.node == data_doc['host'] and self.status == data_doc['status'] and self.file == None and self.set == None:
```

Comment:
With all of this, if you use an editor like PyCharm, you can see how long the lines are.  We try to cap the line length for readability at 80 (up to 120) per line.